cypress/pageFunction/AddCartFunctions.ts

```diff
@@ -0,0 +1,33 @@
+import cypress from 'cypress';
+import { AddCartPageLocators } from '../locators/AddCartPageLocators';
+import { PageTextValues } from '../pageEnums/PageTextValues';
+import { Runnable } from 'mocha';
+export class AddCartFunctions{
+    static selectLocation(){
+        cy.fixture('requiredData').then((data)=>{
+            const location = data.Location;
```

Comment:
Do we need fixture data in this page function class ?

Reply:
@ajinkyaces yeah for reading location value for json but I will change it. I will try to pass it from test class